packages/cxl-ui/src/components/cxl-marketing-nav.js

```diff
@@ -273,7 +280,48 @@ export class CXLMarketingNavElement extends LitElement {
         );
 
         // Populate.
-        contextMenu.items = this._createContextMenuItems(menuItem.children);
+        const contextMenuItems = this._createContextMenuItems(menuItem.children);
+        contextMenu.items = contextMenuItems;
+
+        const setDescription = async () => {
+          contextMenu.items = contextMenuItems;
+          contextMenu.render();
+
+          const description = document.createElement('vaadin-context-menu-item');
+          render(html`${unsafeHTML(menuItem.description)}`, description);
+
+          setTimeout(() => {
+            description.style.maxWidth = getComputedStyle(
+              document.querySelector('vaadin-context-menu-list-box')
+            ).width;
+
+            contextMenu.items = [
+              ...contextMenuItems,
+              { component: document.createElement('hr') },
+              { component: description },
+            ];
+            contextMenu.render();
+          }, 0);
+        };
+
+        if (menuItem.description) {
+          contextMenu.addEventListener('opened-changed', () => {
+            if (!document.querySelector('vaadin-context-menu-list-box')) return;
+            setDescription();
+          });
+
+          const observe = (mediaQueryString, callback) => {
+            const observer = window.matchMedia(mediaQueryString);
+            const matches = (mediaQueryList) => callback(mediaQueryList.matches);
+            matches(observer);
+            observer.addEventListener('change', matches);
+          };
+
+          observe('(min-width: 420px) and (min-height: 420px)', () => {
```

Comment:
This magic number is from upstream vaadin-context-menu internals? We can't refer to it via some symbol?

Reply:
I'm not able to find any symbols.